eng/packages/http-client-csharp/eng/pipelines/ci.yml

```diff
@@ -0,0 +1,67 @@
+trigger: none
```

Comment:
I would put this ci.yml directly in the http-client-csharp directory.

Reply:
updated